api/institutions/serializers.py

```diff
@@ -33,3 +64,27 @@ def get_absolute_url(self, obj):
 
     class Meta:
         type_ = 'institutions'
+
+
+class InstitutionAuthSerializer(ser.Serializer):
```

Comment:
This serializer has no fields. Therefore, it doesn't really do any (de)serialization. You can probably remove this. Instead, you could move this logic to the new `Authentication` policy, as suggested in https://github.com/CenterForOpenScience/osf.io/pull/4774/files#r51460341